package.json

```diff
@@ -26,7 +26,7 @@
     "prepublishOnly": "rm -rf lib && npm run build"
   },
   "peerDependencies": {
-    "@material-ui/core": "^4.9.11",
+    "@material-ui/core": "^4.9.14",
     "@material-ui/lab": "^4.0.0-alpha.50",
```

Comment:
For these versions, @qkdreyer told me that the version we could use are `@material-ui/core@4.9.13` and `@material-ui/lab@1.0.0-alpha.52` (at least, that is what is defined in the dashboard.

Reply:
then we'll need to upgrade the dashboard, it's a major version of lib-ui anyway

Reply:
Actually, https://github.com/rgsystemes/kb/issues/4960#issuecomment-627955266 needs that update